### PR TITLE
Enhancement/fix benchmarks

### PR DESCRIPTION
**Pull request by pawellysy**
Fixed the benchmark comparison scripts.

### CODE REVIEW

**Review by kamil-musialowski — APPROVED** [review]

File: .github/workflows/stock-benchmark.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ on:
       - 'master'
     paths:
       - 'ts/Stock/**'
+      - 'ts/Core/Axis/OrdinalAxis.ts'
+      - 'ts/Extensions/Annotations/**'
```
Comment (pawelfus) [inline .github/workflows/stock-benchmark.yml:9]: + `StockChart.ts` and `BrokenAxis.ts` please

**Review by pawelfus — CHANGES_REQUESTED** [review]
While you are at this.. 😬

**Comment by pawelfus** [comment]
How hard would it be to add a column with %-change? Is 100ms too long? It depends if the test takes 500ms or 5000ms

**Review by DJTechnoo — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]
Thanks!

**Pull request merged**